tests/logictest/suites/crdb/join

```diff
@@ -306,21 +306,9 @@ SELECT * FROM onecolumn NATURAL JOIN twocolumn;
 42 53
 
 statement query IIII
-SELECT * FROM twocolumn AS a JOIN twocolumn AS b ON a.x = a.y;
```

Comment:
Is this correct? The result set changes.

Reply:
The original result is wrong, the ticket caught the error. Now the result is correct.